lib/adapters/travis.js

```diff
@@ -64,12 +64,15 @@ async function getTotalBuilds(user, repo, auth) {
   return resJson['@pagination'].count;
 }
 
-async function fetchPipelines(buildsDir, user, repo, auth, concurrency, downloadHook) {
+async function fetchPipelines(buildsDir, user, repo, auth, concurrency, since, downloadHook) {
```

Comment:
I think we should think how we are passing all of these options around (it's a ton of params) but I didnt want to make too many changes at once

Reply:
Agree, let's take a look at it later. I am working on refactor right now. To test and release a new version.